test/test_Data.py

```diff
@@ -47,8 +49,7 @@ def test_Loader(self, utils, insert_data_from_csv):
         self.assertEqual(len(data), utils.count_bboxes())
 
     def test_LoaderDescriptorsImages(self, utils, insert_data_from_csv):
-        classes = ["_Image", "_Descriptor", "_DescriptorSet"]
-        self.remove_entities(utils, classes)
+        utils.remove_all_objects()
```

Comment:
```suggestion
        self.assertTrue(utils.remove_all_objects())
```
This is valid for all uses of remove_all_objects()